packages/design-system/src/components/FieldContainer/FieldContainer.tsx

```diff
@@ -60,16 +69,22 @@ interface FieldContainerProps {
 }
 
 export class FieldContainer extends React.Component<FieldContainerProps> {
+  static defaultProps = {
+    errorPlacement: 'top'
+  }
+  
   constructor(props: FieldContainerProps) {
     super(props);
 
     this.id = props.id || uniqueId('field_');
-    this.labelId = props.labelId || uniqueId('field_label_');
+    this.labelId = props.labelId || `${this.id}-label`;
```

Comment:
I updated the generated `ids` to be more readable, i.e. `field_1`, `field_1-label`, `field_1-error` instead of `field_1`, `field_label_2`, `field_error_3`.